### ISSUE TITLE
date front matter value (2023-12-08 17:02:02) is invalid

### ISSUE THREAD

**Issue by wangzhankun** [oleeskild/digitalgarden#236]
PROBLEM AS TITLE MENTIONED.

Build Log:
```log
<pre class="c_rw c_bv c_nd c_jj c_ux c_uy c_dd c_vl c_vm c_vn c_vo c_gf c_gg c_uz c_bx c_vp" style="box-sizing: border-box; border-radius: 0px; color: rgb(49, 49, 49); display: block; font-size: 12px; margin: 0px; overflow: auto; padding: 16px 32px; width: 1262.67px; background-color: rgb(242, 242, 242); border-width: 0px 0px 1px; border-style: solid; border-color: rgb(217, 217, 217); border-image: initial; font-family: monaco, courier, monospace; height: 500px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
17:32:39.681 | Cloning repository...
-- | --
17:32:40.781 | From https://github.com/wangzhankun/knowledge4public
17:32:40.782 | * branch            0c98fbf5e6f5409966b860157e87a16c743dc04a -> FETCH_HEAD
17:32:40.782 |  
17:32:40.915 | HEAD is now at 0c98fbf Add content src/site/notes/BrowserClip/Linux 设备驱动之 UIO 机制 - allcloud - 博客园.md
17:32:40.916 |  
17:32:41.032 |  
17:32:41.067 | Success: Finished cloning repository files
17:32:41.908 | Detected the following tools from environment: npm@9.6.7, nodejs@18.17.1
17:32:41.909 | Installing project dependencies: npm clean-install --progress=false
17:32:53.462 |  
17:32:53.462 | added 493 packages, and audited 494 packages in 11s
17:32:53.463 |  
17:32:53.463 | 119 packages are looking for funding
17:32:53.463 | run `npm fund` for details
17:32:53.468 |  
17:32:53.468 | 4 vulnerabilities (2 moderate, 2 high)
17:32:53.468 |  
17:32:53.469 | To address issues that do not require attention, run:
17:32:53.469 | npm audit fix
17:32:53.469 |  
17:32:53.469 | Some issues need review, and may require choosing
17:32:53.469 | a different dependency.
17:32:53.469 |  
17:32:53.469 | Run `npm audit` for details.
17:32:53.490 | Executing user command: npm run build
17:32:54.417 |  
17:32:54.418 | > web@1.0.0 build
17:32:54.418 | > npm-run-all get-theme build:*
17:32:54.418 |  
17:32:55.047 |  
17:32:55.048 | > web@1.0.0 get-theme
17:32:55.048 | > node src/site/get-theme.js
17:32:55.048 |  
17:32:55.892 |  
17:32:55.893 | > web@1.0.0 build:eleventy
17:32:55.893 | > cross-env ELEVENTY_ENV=prod NODE_OPTIONS=--max-old-space-size=4096 eleventy
17:32:55.894 |  
17:32:57.686 | [11ty] Problem writing Eleventy templates: (more in DEBUG output)
17:32:57.687 | [11ty] date front matter value (2023-12-08 17:02:02) is invalid for ./src/site/notes/BrowserClip/Linux 设备驱动之 UIO 机制 - allcloud - 博客园.md (via Error)
17:32:57.687 | [11ty]
17:32:57.687 | [11ty] Original error stack trace: Error: date front matter value (2023-12-08 17:02:02) is invalid for ./src/site/notes/BrowserClip/Linux 设备驱动之 UIO 机制 - allcloud - 博客园.md
17:32:57.688 | [11ty]     at Template.getMappedDate (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/Template.js:968:15)
17:32:57.688 | [11ty]     at Template.addPageDate (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/Template.js:403:30)
17:32:57.688 | [11ty]     at Template.getData (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/Template.js:390:29)
17:32:57.688 | [11ty]     at async TemplateMap.add (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/TemplateMap.js:65:16)
17:32:57.688 | [11ty]     at async Promise.all (index 7)
17:32:57.688 | [11ty]     at async TemplateWriter._createTemplateMap (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/TemplateWriter.js:325:5)
17:32:57.689 | [11ty]     at async TemplateWriter.generateTemplates (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/TemplateWriter.js:360:5)
17:32:57.689 | [11ty]     at async TemplateWriter.write (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/TemplateWriter.js:407:23)
17:32:57.689 | [11ty]     at async Eleventy.executeBuild (/opt/buildhome/repo/node_modules/@11ty/eleventy/src/Eleventy.js:1191:13)
17:32:57.689 | [11ty] Copied 63 files / Wrote 0 files in 0.33 seconds (v2.0.1)
17:32:57.748 | ERROR: "build:eleventy" exited with 1.
17:32:57.765 | Failed: Error while executing user command. Exited with error code: 1
17:32:57.775 | Failed: build command exited with code: 1
17:33:03.691 | Failed: error occurred while running build command

</pre>
```

**Comment by SheepYY039** [comment]
I created a pull request (#252) which fixes this issue :)

**Comment by oleeskild** [comment]
Fixed in version 1.61.0
